internal/api/v1/models/app.go

```diff
@@ -62,6 +64,10 @@ func (ar *AppRef) EnvSecret() string {
 	return ar.Name + "-env"
 }
 
+func (ar *AppRef) PVCName() string {
+	return fmt.Sprintf("%s-%s", ar.Org, ar.Name)
```

Comment:
This apparently generates the name of a kube resource, i.e. of a `PVC`.
Might need a prefix. Will, if an org name like `12monkeys` is allowed.
See #690 for analogous issues.
Certainly needs `names.GenerateResourceName` from #700 to ensure the length limit.
Note, the test's `k get pvc ...` has to be modified to match.

Reply:
`12monkeys` namespace (org) is allowed and so does the `12monkeys.sample` PVC. I used `names.GenerateResourceName` helper (https://github.com/epinio/epinio/pull/701/commits/4d8e8b0bb5bf2db53ffa6e6d15bc493e652bad63)

Reply:
Ok